modules/calendar.md

```diff
@@ -98,6 +98,7 @@ config: {
 | --------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `url`                 | The url of the calendar .ical. This property is required. <br><br> **Possible values:** Any public accessible .ical calendar.                                                                                                                                   |
 | `symbol`              | The symbol to show in front of an event. This property is optional. <br><br> **Possible values:** See [Font Awesome](https://fontawesome.com/v5.15/icons?m=free) website. To have multiple symbols you can define them in an array e.g. `["calendar", "plane"]` |
+| `symbolClassName` | The Font Awesome class name. This property is optional. <br><br>The default value is `fas fa-fw fa-`. |
```

Comment:
Thanks, maybe add an example for which icon its needed?

Reply:
Agreed, added a better description with an example now 👍